met-api/migrations/versions/c656f3f82334_add_languages_and_tenant_mapping.py

```diff
@@ -0,0 +1,42 @@
+"""
+Add the list of supported languages and create a table to map them to tenants when selected.
+
+Revision ID: c656f3f82334
+Revises: 22fb6b5b5aed
+Create Date: 2024-05-28 15:45:56.151488
+
+"""
+from alembic import op
+import sqlalchemy as sa
+import json
+
+
+# revision identifiers, used by Alembic.
+revision = 'c656f3f82334'
+down_revision = '22fb6b5b5aed'
+branch_labels = None
+depends_on = None
+
+with open("migrations/supported_languages.json", "r") as read_file:
```

Comment:
Has maintainability been considered here? What will it look like when we need to add or remove a language? Can we still use the same JSON file or will we be manually migrating to match any edits made?

Reply:
Great point!

Adding/removing languages in the future would be done with a DB migration. I don't expect us to have to do so that often. Given that, maybe it would be better to rename `supported_languages` to `initial_languages` or something, knowing that it may not be the exhaustive in the long term.

I could have also not had a separate file but I felt the size of the migration file would be super huge and less readable that way. Do you think maybe I should just move the list into the migration file given that they'll be the ultimate source of truth?